test/system-libraries/src/namespaces/a/codegen/libraries/ASystemLib.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+/* Autogenerated file. Do not edit manually. */
+
+import { ASystem } from "../../ASystem.sol";
+import { revertWithBytes } from "@latticexyz/world/src/revertWithBytes.sol";
+import { IWorldCall } from "@latticexyz/world/src/IWorldKernel.sol";
+import { SystemCall } from "@latticexyz/world/src/SystemCall.sol";
+import { Systems } from "@latticexyz/world/src/codegen/tables/Systems.sol";
+import { ResourceId } from "@latticexyz/store/src/ResourceId.sol";
+import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
+
+type ASystemType is bytes32;
+
+ASystemType constant aSystem = ASystemType.wrap(0x737961000000000000000000000000004153797374656d000000000000000000);
```

Comment:
should we render a comment above this that explains the hex? (ie `// equivalent to WorldResourceIdLib.encode({ 
typeId: RESOURCE_SYSTEM, namespace: "a", "name: "ASystem" }))`